xdggs/h3.py

```diff
@@ -22,6 +21,32 @@
 from xdggs.utils import _extract_cell_id_variable, register_dggs
 
 
+def polygons_shapely(wkb):
+    import shapely
+
+    return shapely.from_wkb(wkb)
+
+
+def polygons_geoarrow(wkb):
+    import shapely
+    from arro3.core import list_array
+
+    polygons = shapely.from_wkb(wkb)
+    _, coords, (geom_offsets, ring_offsets) = shapely.to_ragged_array(polygons)
```

Comment:
Nit: I'd assert the geometry type (first returned element of `to_ragged_array`) matches `Polygon` here